src/formBuilder/CardGeneralParameterInputs.js

```diff
@@ -243,6 +248,48 @@ export default function CardGeneralParameterInputs({
           />
         </div>
       </div>
+
+      <div
+        className={classnames('card-entry', {
+          'wide-card-entry': !showObjectNameInput,
+        })}
+      >
+        <h5>
+          {`${inputColumnSize} `}
+          <Tooltip
+            text={
+              mods &&
+              mods.tooltipDescriptions &&
+              typeof mods.tooltipDescriptions.inputColumnSize === 'string'
+                ? mods.tooltipDescriptions.inputColumnSize
+                : 'The size of form input displayed on the form'
+            }
+            id={`${(keyState: string)}-inputinfo`}
+            type='help'
+          />
+        </h5>
+        <Select
```

Comment:
Why is the column size a dropdown rather than a (positive) number input?

Reply:
I've changed it to a number input and removed default in order to make it 100% customisable.